src/Engine/UnitActionType.java

```diff
@@ -410,7 +412,7 @@ public GameActionSet getPossibleActions(GameMap map, Path movePath, Unit actor,
       XYCoord moveLocation = new XYCoord(movePath.getEnd().x, movePath.getEnd().y);
       if( ignoreResident || map.isLocationEmpty(actor, moveLocation) )
       {
-        return new GameActionSet(new GameAction.ExplodeAction(actor, movePath, this), false);
+        return new GameActionSet(new GameAction.ExplodeAction(actor, movePath, this), true); // We don't really need a target, but I want a confirm dialogue
```

Comment:
This feels like a bit of a hack.
I'm thinking we probably want to move whether an action is targeted to the UnitActionType itself?
I almost did it, but then there was too many ripples and I'm lazy.

Reply:
That sounds sensible in the 10 seconds I've considered it so far. Out of scope for this PR though, yes.